src/types/submissions.ts

```diff
@@ -57,6 +57,8 @@ export type FormSubmission = DraftSubmission &
     previousFormSubmissionApprovalId?: string
     /** The id of the scheduled task being completed */
     taskId?: number
+    /** The name of the scheduled task being completed */
+    taskName?: string
```

Comment:
prob need the group name and instance label as well.

Reply:
Caaaaan do